tests/check_merge_conflict_test.py

```diff
@@ -9,21 +9,25 @@
 from testing.util import git_commit
 
 
-@pytest.fixture
-def f1_is_a_conflict_file(tmpdir):
+@pytest.fixture(scope='function', params=['repo', 'worktree'])
+def f1_is_a_conflict_file(tmpdir, request):
```

Comment:
I would prefer not to use parametrized fixtures -- they're really difficult to understand and maintain

what you can actually do here instead is write a separate test which consumes this fixture and the makes a worktree from that and call the original test -- it's going to be a lot easier to follow than this action-at-a-distance